pygtc/pygtc.py

```diff
@@ -1095,7 +1104,11 @@ def __plot1d(ax, nChains, chains1d, weights, nBins, smoothingKernel,
         # Line for hidden histogram
         for k in reversed(range(nChains)):
             if plotData[nChains-1-k] is not None:
-                ax.plot(plotData[nChains-1-k][0], plotData[nChains-1-k][1],
+                if linewidth is not None:
```

Comment:
This if/else test should get replaced with passing `lw=linewidth or 1` into the plot method. This will work for all cases _except_ if the user passes in `linewidth=0`, which I'd argue is not really a useful choice in any case and therefor doesn't need to be supported (and in fact _should_ be active suppressed!).